05_stringer/NimishMudhale/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+	// When
+	main()
+	// Then
+	expected := fmt.Sprint("127.0.0.1")
+	actual := buf.String()
+	r.Equalf(expected, actual, "Unexpected output in main()")
+}
+
+var cases = []struct {
+	in  IPAddr
+	out string
+}{
+	{IPAddr{10, 0, 0, 0}, "10.0.0.0"},
+	{IPAddr{100, 100, 100, 0}, "100.100.100.0"},
+	{IPAddr{0, 0, 0, 0}, "0.0.0.0"},
+	{IPAddr{110, 52, 32, 31}, "110.52.32.31"},
```

Comment:
Try out `IPAddr{1, 2}` and `IPAddr{}` as well, see what happens.

Reply:
done